modules/update-best-practices.adoc

```diff
@@ -36,6 +36,16 @@ In most cases this alert does not block patch updates, but you cannot perform a
 
 For more information about the `Upgradeable` condition, see "Understanding cluster Operator condition types" in the additional resources section.
 
+[id="sdn-support-removal"]
+== SDN support removal
+
+OpenShift SDN network plugin was deprecated in versions 4.15 and 4.16. With this release, the SDN network plugin is no longer supported and the content has been removed from the documentation.
+
+[IMPORTANT]
+====
+It is not possible to update a cluster to {product-title} 4.17 if it is using the OpenShift SDN network plugin. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17. 
```

Comment:
Is there a way to link back up to [the 4.17 release note about this][1], in case that release note grows a link to [the 4.16 release note][2] and/or [the 4.16 migration process][3]?

[1]: https://docs.openshift.com/container-platform/4.17/release_notes/ocp-4-17-release-notes.html#ocp-4-17-sdn-plugin_release-notes
[2]: https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-networking-openshift-sdn-upgrade-block_release-notes
[3]: https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#migrate-from-openshift-sdn

Reply:
Yes - @JoeAldinger doesn't have the updated release note yet. Right now, we have https://github.com/openshift/openshift-docs/pull/81504/files. Joe mentioned that he is working through that dilemma now. I'll check in with him to see what we can do. I agree it would be best to also provide a reference on how to perform the migration :)

Reply:
cc @stevsmit and @dfitzmau We need to connect with Olivia on how to reference this in the 4.17 RNs. I'm not sure what is best practice here. We could move it to the Dep/Rem tables with a possible footnote that links (can't xref because of the branch dilemma) to the migration process on 4.16 🤔

Reply:
Should this module also be removed from `main`? `Main` and the `enterprise-4.17` branches should still be in sync.

Reply:
Linking to a release note document is difficult given how version numbers change on each branch. I think linking to a 4.16 migration document would be easier. The 4.17 RN doc has a note for the removal of the SDN plugin:

https://docs.openshift.com/container-platform/4.17/release_notes/ocp-4-17-release-notes.html#ocp-4-17-sdn-plugin_release-notes

We could expand information here about migration?

Reply:
Moving to Slack for more conversation

Reply:
Decided to add a reference to docs.redhat.com!